app/test/max_total_blob_size_test.go

```diff
@@ -105,16 +86,6 @@ func (s *MaxTotalBlobSizeSuite) TestSubmitPayForBlob_blobSizes() {
 			require.NoError(t, err)
 			require.NotNil(t, res)
 			require.Equal(t, tc.want, res.Code, res.Logs)
-
-			sq, err := square.Construct([][]byte{blobTx}, appconsts.LatestVersion, squareSize)
-			if tc.want == abci.CodeTypeOK {
-				// verify that if the tx was accepted, the blob can fit in a square
-				assert.NoError(t, err)
-				assert.False(t, sq.IsEmpty())
-			} else {
-				// verify that if the tx was rejected, the blob can not fit in a square
-				assert.Error(t, err)
```

Comment:
[blocking]
do we still want to keep this assertion?

Reply:
Good catch. I elected to remove b/c it seems like this test is covering behavior of something unrelated (square construction).

I brought back the test in: 3e0c90216e81eeb52a3fa91e7044d97d0fe98da6